mongo/baixaEstoque.js

```diff
@@ -0,0 +1,8 @@
+use("ecomm")
+
+let decrescimoEmEstoque = db.products.updateOne(
+  {$and: [{NOME:"Galaxy Tab S8"}, {ESTOQUE:{$gt: 0}}]},
+  {$inc: {ESTOQUE: -2}}
```

Comment:
O que aconteceria nesse cenário se o valor de estoque for igual a 1?

Reply:
Não é possível encontrar o documento, então o estoque nunca zera (ou fica negativo, no caso).